Formula/brew-php-switcher.rb

```diff
@@ -0,0 +1,20 @@
+class BrewPhpSwitcher < Formula
+  desc "Switch Apache / Valet / CLI configs between PHP versions"
+  homepage "https://github.com/philcook/php-switcher"
+  url "https://github.com/philcook/brew-php-switcher/archive/v2.0.tar.gz"
+  sha256 "c2303b1b1a66ee90ed900c3beabacd6aa4e921dbcad5242e399c45d86899bc88"
+
+  bottle :unneeded
+
+  depends_on "php" => :test
+
+  def install
+    bin.install "phpswitch.sh"
+    bin.install_symlink "phpswitch.sh" => "brew-php-switcher"
+  end
+
+  test do
+    # Ensure that PHP version actually switches
+    system "#{bin}/brew-php-switcher | grep 'usage: brew-php-switcher version \\[-s\\|-s=\\*\\] \\[-c=\\*\\]'"
```

Comment:
As mentioned earlier: use `assert_match` (to match `"usage: brew-php-switcher version "` that some `shell_output` from brew-php-switcher is as expected (as systems exit codes do not seem to be useful).

Reply:
Sorry for the misunderstanding. I'm not familiar with how to do all of this with Homebrew/Ruby as I'm mainly a PHP developer. I think it should fit with your expectation now.